docs/users-guide/05-visualizing-results.md

```diff
@@ -208,7 +208,7 @@ There's quite a bit you can do in this tab:
 
 There are three main things you can do here:
 
-- Change the scale for your axes. If you're looking at a time series chart, your x-axis can use a time series scale or an ordinal one. Your y-axis can use a linear, power, or logarithmic scale.
+- Change the scale for your axes. If you're looking at a time series chart, your x-axis can use a time series scale or an ordinal one. When using "Timeseries", it will always be displayed in ascending order, so oldest to newest, while "Ordinal" will display in the order the data is returned. Your y-axis can use a linear, power, or logarithmic scale.
```

Comment:
```suggestion
- Change the scale for your axes. If you're looking at a time series chart, your x-axis can use a time series scale or an ordinal one. When using "Time series", it will always be displayed in ascending order, so oldest to newest, while "Ordinal" will display in the order the data is returned. Your y-axis can use a linear, power, or logarithmic scale.
```

Reply:
I know it's "time series", but our option is called "Timeseries" - can we keep this as is? It will also help for people who are searching